build_tools/circle/check_pypy_marker.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -x
+set -e
+
+# Check for pypy marker in the commit message and
+# trigger the job if the marker is present.
+
+commit_msg=$(git log --format=%B -n 1 $CIRCLE_SHA1)
+
+if [[ "$commit_msg" =~ "[pypy]" ]]
```

Comment:
May I suggest to use `"[ci pypy]"` instead to make it explicit that this is about Continuous Integration?